CMake/nx.cmake

```diff
@@ -0,0 +1,12 @@
+message(STATUS "Using 32-bit toolchain")
```

Comment:
I don't think this file is necessary

Reply:
Yeah I was thinking the same, but wanted things to build before removing it :)